packages/fx-core/templates/plugins/resource/aad/auth/bot/js/sso/teamsSsoBot.js

```diff
@@ -1,3 +1,7 @@
+// This file implements a `TeamsActivityHandler` class using `ssoDialog` and add `showUserInfo` as a command.
+// See https://docs.microsoft.com/en-us/javascript/api/botbuilder/teamsactivityhandler?view=botbuilder-ts-latest for more about `TeamsActivityHandler`.
```

Comment:
One suggestion might to remove locale information `en-us` when referencing doc site, for better localization support.